installer/templates/phx_assets/webpack.config.js

```diff
@@ -46,8 +45,7 @@ module.exports = (env, options) => {
     },
     plugins: [
       new MiniCssExtractPlugin({ filename: '../css/app.css' }),
-      new CopyWebpackPlugin([{ from: 'static/', to: '../' }])
+      new CopyWebpackPlugin({ patterns: [{ from: "static/", to: "../" }] }),
```

Comment:
I know it doesn’t _really_ matter, but the rest of the file is using single quotes.

Reply:
Good catch. I fixed it in https://github.com/phoenixframework/phoenix/pull/4146/commits/7dfc362fdf9e4f54d5b1c6f8e29914739d7f12c4